Plugins/Other/aman-config.json

```diff
@@ -23,8 +49,18 @@
             { "source": "remainingDistance", "width": 4, "rightAligned": true },
             { "source": "static", "width": 1 },
             { "source": "directRouting", "width": 5, "rightAligned": true, "defaultValue": "-----" },
+            { "source": "static", "width": 1 }
+        ],
+        "SinglerunwayLayout": [
+            { "source": "callsign", "width": 8 },
+            { "source": "static", "defaultValue": "�", "width": 2, "isViaFixIndicator": true },
```

Comment:
Sjekk defaultValue her. 
Kommer � med UTF-8 encoding
Kommer Ī med ISO 8859-10 encoding.

Antar det skal være noe annet?

Reply:
Det skal være ¤

Reply:
Fila må være i ANSI/Windows-1252 for å lese defaultvalue riktig. Oppdatert i pullrequesten nå